validate.py

```diff
@@ -8,125 +8,126 @@ def __init__(self, filepath):
             raise "Dont Found Format"
 
-class Dataset_Fields_Validator(object):
+
+class DatasetFieldsValidator(object):
 
     def __init__(self, dataset):
-        self.dataset = dataset
+        self.field = dataset["fields"]
 
     def verify_fields(self):
-        if 'fields' in self.dataset:
-            self.field = dataset.get('fields')
-            body =  self.field
-            for body_field in body:
+        if 'fields' in dataset:
+            for body_field in self.field:
                 if 'name' in body_field:
-                    if isinstance(body_field.get('name'), str):
-                        self.field_name = body_field.get('name')
+                    self.name = body_field.get('name')
+                    if isinstance(self.name, str):
+                        return True
                     else:
                         raise "Name type must be string"
                 else:
                     raise "Dont Found Name in Fields"
                 if 'generator' in body_field:
-                    if isinstance(body_field.get('generator'), dict):
-                        self.field_generator = body_field.get('generator')
+                    self.generator = body_field.get('generator')
+                    if isinstance(self.generator, dict):
+                        return True
                     else:
                         raise "Generator must be dict"
                 else:
                     raise "Dont Found Generator in Fields"
                 if 'type' in body_field:
-                    if isinstance(body_field.get('type'), (str, bool, float, int, list)):
-                        self.field_generator = body_field.get('type')
+                    self.type = body_field.get('type')
+                    if isinstance(self.type, (str, bool, float, int, list)):
+                        return True
                     else:
                         raise "Type is not "
                 else:
                     raise "Dont Found Type in Fields"
         else:
-            raise "Dont Found Fields"
+            raise "Fields Dont Exist in Dataset"
 
-class Dataset_Serializer_Validator(object):
+
+class DatasetSerializerValidator(object):
 
     def __init__(self, dataset):
-        self.dataset = dataset
+        self.serializers = dataset["serializers"]
+        self.to = self.serializers["to"]
 
     def verify_serializers(self):
-        if 'serializers' in self.dataset:
-            if 'to' in self.dataset.get('serializers'):
-                self.serializer_to = dataset.get('serializers').get('to')
-                body_to = self.serializer_to 
-                for dict_to in body_to:
-                    if 'type' in dict_to:
-                        if isinstance(dict_to.get('type'), str):
-                            self.serializer_type = dict_to.get('type')
-                        else:
-                            raise "Serializer type must be string"
+        if 'to' in self.serializers:
```

Comment:
You already get the "to" property in the __init__ method, why you checking again in this statement?